nuxt.config.ts

```diff
@@ -44,14 +48,5 @@ export default defineNuxtConfig({
       display: "standalone",
       crossorigin: "use-credentials",
     },
-    // workbox: {
-    //   runtimeCaching: [
-    //     {
-    //       urlPattern: 'https://www.datocms-assets.com/.*',
-    //       handler: 'cacheFirst',
-    //       method: 'GET',
-    //     },
-    //   ],
-    // },
```

Comment:
Leftovers? If these are todos, let's create trello tickets for them.

Reply:
yep, ok